exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateMissingTablesAndColumnsTests.kt

```diff
@@ -698,10 +698,18 @@ class CreateMissingTablesAndColumnsTests : DatabaseTestsBase() {
 
         // SQLite does not recognize creation of schema other than the attached database
         withDb(excludeSettings = listOf(TestDB.SQLITE)) { testDb ->
+            // Should not require to be in the same schema
             SchemaUtils.createSchema(schema)
             SchemaUtils.create(parentTable, childTable)
```

Comment:
To resolve the SQL Server issue, please provide an authentication/owner to the `Schema` object, which is necessary when setting a new schema. This can be done by moving instantiation into the `withDb` block, with something like this:
```kt
val schema = if (testDb == TestDB.SQLSERVER) {
    Schema(schemaName, "guest")
} else {
    Schema(schemaName)
}
```